types/node-fetch/index.d.ts

```diff
@@ -102,7 +102,7 @@ export class Headers implements Iterable<[string, string]> {
     get(name: string): string | null;
     getAll(name: string): string[];
     has(name: string): boolean;
-    raw(): { [k: string]: string[] | string | undefined };
+    raw(): { [k: string]: string[] | string };
```

Comment:
Can you drop the `string` and leave only `string[]`. Looking at the code (see links in my earlier comment) the internal headers map, which `raw` returns verbatim, will always have values of type `string[]`.

Reply:
Good point, fixed now!